src/components/table/_macro.njk

```diff
@@ -17,55 +18,111 @@
+                                {% if th.colspan %}colspan="{{ th.colspan }}"{% endif %}
+                                {% if th.rowspan %}rowspan="{{ th.rowspan }}"{% endif %}
+                                {% if 'sortable' in variants %}aria-sort="{{- th.ariaSort | default('none') -}}"{% endif %}
+                                {% if th.widthPercentage %}width="{{ th.widthPercentage }}%"{% endif %}
+                            >
+                                <span class="ons-table__header-text">{{- th.value | safe -}}</span>
+                                {% if 'sortable' in variants %}
+                                    {{-
+                                        onsIcon({
+                                            "iconType": "sort-sprite",
+                                            "id": th.value | replace(' ', '-'),
+                                            "classes": 'ons-u-d-no'
+                                        })
+                                    -}}
+                                {% endif %}
+                            </th>
+                        {% endfor %}
+                    </tr>
+                {% endif %}
             </thead>
             <tbody class="ons-table__body">
                 {% for tr in params.trs %}
-                    <tr class="ons-table__row{{ ' ons-table__row--highlight' if tr.highlight }}" {% if tr.id %}id="{{ tr.id }}"{% endif %}>
+                    {% set trIndex = loop.index0 %}
+                    {% if rowspanRemaining > 0 %}
+                        {% set previousRowHasRowspan = true %}
+                        {% set rowspanRemaining = rowspanRemaining - 1 %}
+                    {% else %}
+                        {% set previousRowHasRowspan = false %}
+                        {% set currentRowspan = tr.tds[0].rowspan | default(1) %}
+                        {% if currentRowspan > 1 %}
+                            {% set rowspanRemaining = currentRowspan - 1 %}
+                        {% endif %}
+                    {% endif %}
+                    <tr
+                        class="ons-table__row{{ ' ons-table__row--highlight' if tr.highlight }}{{ ' ons-table__row--after-rowspan' if previousRowHasRowspan }}"
+                        {% if tr.id %}id="{{ tr.id }}"{% endif %}
+                    >
                         {% for td in tr.tds %}
-                            <td
-                                class="ons-table__cell{{ ' ' + td.tdClasses if td.tdClasses else '' }}{{ ' ons-table__cell--numeric' if td.numeric }}"
-                                {% if td.id %}id="{{ td.id }}"{% endif %}
-                                {% if td.data %}{{ ' ' }}data-th="{{ td.data }}"{% endif %}
-                                {% if td.dataSort %}{{ ' ' }}data-sort-value="{{ td.dataSort }}"{% endif %}
+                            {% if td.heading %}
```

Comment:
```suggestion
                            {% if td.heading == true %}
```
At the moment this defaults to true if something other than a boolean is passed in. I think it should probably default to false. This change will mean it does that